modules/apps/roles/roles-test/src/testIntegration/java/com/liferay/roles/service/test/RoleServiceTest.java

```diff
@@ -104,9 +175,16 @@ protected String getRoleName() {
 		return RoleConstants.USER;
 	}
 
+	private ClassName _className;
+
+	@Inject
+	private ClassNameLocalService _classNameLocalService;
+
 	@DeleteAfterTestRun
 	private Role _role;
 
+	private final List<Role> _roles = new ArrayList<>();
```

Comment:
Is it possible for us to use the `@DeleteAfterTestRun` annotation here instead of manually removing them in `tearDown`?

Reply:
This deletion has to be manually handled in `tearDown` because I need to make sure `className` gets deleted after the roles are removed. Otherwise, we run into exceptions

Reply:
That may warrant a code comment so Brian doesn't SF it, or at least point it out to him in the PR.

Reply:
Will do.